docs/install/install_everest_helm_charts.md

```diff
@@ -0,0 +1,153 @@
+
+        1. Run the following command to change the Everest service type to `NodePort`:
+
+            ```sh
+            kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "NodePort"}}
+            ```
+        2. The following command displays the port assigned by Kubernetes to the everest service, which is `32349` in this case.
+
+            ```sh
+            kubectl get svc/everest -n everest-system
+            NAME      TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+            everest   NodePort   10.43.139.191   <none>        8080:32349/TCP   28m
+            ```
+
+        3. Retrieve the external IP addresses for the kubernetes cluster nodes.
+
+            ```sh
+            kubectl get nodes -o wide
+            NAME                   STATUS   ROLES    AGE   VERSION             
+            INTERNAL-IPEXTERNAL-IP  OS-IMAGE                        KERNEL-VERSION   
+            CONTAINER-RUNTIME
+            gke-everest-test-default-pool-8bbed860-65gx   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.199   34.175.155.135   Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            gke-everest-test-default-pool-8bbed860-pqzb   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.200   34.175.120.50    Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            gke-everest-test-default-pool-8bbed860-s0hg   Ready    <none>   3m35s   
+            v1.30.3-gke.1969001   10.204.15.201   34.175.201.246   Container- 
+            Optimized OS from Google   6.1.100+         containerd://1.7.19
+            ```
+        
+        4. To launch the Percona Everest UI and create your first database cluster, go to the IP address/port found in steps 2 and 3. In this example, the external IP address used is [http://34.175.155.135:32349](http://34.175.155.135:32349). Nevertheless, you have the option to use any node IP specified in the above steps.
+
+    === "Port Forwarding"
+        Run the following command to use `Kubectl port-forwarding` for connecting to Everest without exposing the service:
+                
+        ```sh
+        kubectl port-forward svc/everest 8080:8080 -n everest-system
+        ``` 
+
+        To launch the Percona Everest UI and create your first database cluster, go to your localhost IP address [http://127.0.0.1:8080](http://127.0.0.1:8080).
+
+
+5. The default username for logging into the Everest UI is `admin`. You can set a different default admin password by using the `server.initialAdminPassword` parameter during installation.
+
+    The default admin password is stored in plain text. It is highly recommended to update the password using `everestctl` to ensure that the passwords are hashed.
```

Comment:
We need a link to the user management section.

Reply:
Fixed.